the-way-we-work.md

```diff
@@ -0,0 +1,664 @@
+Actions & Behaviours are the things we expect developers to do within their areas of responsibility.
+
+We believe that if developers do these things, and if they try to always improve how they do them, then surely excellent outcomes will follow. 
+
+Note that according to the responsibility matrix above, only some of the below actions and behaviours will be appropriate for each role.
+
+TODO: This section should be "WHAT and HOW OFTEN". Move details to Tools & Methods
+
+TODO: Alternative approach: Combine into "Actions & Behaviours, Tools & Methods" or "Behaviours & Methods" etc.
+
+### 3.1 Code
+
+**Feature Development**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- attend sprint planning at the beginning of each sprint
+- complete at least one Jira ticket per sprint and change its status to "Done"
+- pair with other developers where possible
+- commit code which is deployed to production at least once per sprint
+- attend a retrospective at the end of each sprint
+
+**System Design and Documentation**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- use your technical expertise to lead the design of the solution
+- ensure the system is sufficiently documented
+- ensure the solution is delivered incrementally, with a deployment each sprint
+- ensure the solution is delivered in a way that maximises the involvement and learning of other team members
+- provide specialist technical advice to other team members
+
+**Engineering Quality Framework**
+
+This work will be carried out in collaboration with SDMT.
+
+You will:
+
+- maintain and develop technical expertise in the EQF areas
+- design and deliver technical prototypes or experiments to inform decisions and future development
+- collaborate with the leadership team to decide the future technical direction of NHSUK
+
+### 3.2 Leadership and Autonomy
+
+**20% Time**
+
+You will:
+
+- agree 3 suitable personal objectives and corresponding key results with your line manager
+- ensure that one of these objectives is related to an agreed training activity
+- spend one day each week working on your personal objectives
+- enter your objectives and key results on the corporate PDR system
+- update the corporate PDR system with progress against your objectives and key results
+- ensure Jira tickets are created for these objectives
+- select at least one of these Jira tickets to work on in each sprint
+- ensure this ticket(s) are included in the sprint backlog
+
+**SDMT Operational Planning**
+
+This work will be carried out in collaboration with SDMT.
+
+You will:
+
+- attend the Senior Developers Management Team (SDMT) meeting each week
+- chair SDMT on a rotating basis, preparing the agenda and writing meeting minutes
+- share important developments in your team or area of work
+- raise and discuss problems that require cross-team collaboration to solve
+- offer your assistance to help solve cross-team problems
+- help to maintain the SDMT Plan
```

Comment:
maintain coding standards documentation and ensure its followed by everyone

Reply:
I will add in "maintain coding standards documentation" here and "ensure code complies with our coding standards" as part of a bit about code reviews in section 4.